lib/product/index.js

```diff
@@ -1,47 +1,53 @@
 'use strict';
 
-const _       = require('lodash'),
-      sqlite3 = require('sqlite3'),
-      db      = new sqlite3.Database('prod.db');
+var _       = require('lodash'),
+    sqlite3 = require('sqlite3'),
+    cache   = require('../../cache'),
```

Comment:
I could have used,  exports NODE_PATH=$NODE_PATH:./[foldername] or written a function to build an absolute path. 

And yes I should have added .DS_STORE to the .gitignore file (gotta love Apple). And I also should have run npm shrinkwrap after installing additional dependencies.